EasyProject/src/main/java/org/example/task1_case1/Main.java

```diff
@@ -0,0 +1,138 @@
+package org.example.task1_case1;
+
+import java.util.HashMap;
+import java.util.InputMismatchException;
+import java.util.Map;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+
+
+        int col = 3, row = 3;
```

Comment:
if this game can be only 3x3 then hardcode it

Reply:
done